cleanIntegrationTests.sh

```diff
@@ -2,6 +2,7 @@
 
 docker container stop $(docker container ls -qf name=bitcore_*)
 docker container rm $(docker container ls -aqf name=bitcore_*)
-docker image rm bitcore_test_runner
-docker image rm bitcore_rippled
-/home/kjoseph/dev/bitcore/packages/bitcore-client/bin/wallet-delete --name EthereumWallet-Ci
+docker image rm bitcore-test_runner
+docker image rm bitcore-rippled
+$HOME/dev/bitcore/packages/bitcore-client/bin/wallet-delete --name EthereumWallet-Ci
```

Comment:
Good good. We could probably make this even better and not assume bitcore is in $HOME/dev by replacing `$HOME/dev/bitcore` with `$(dirname "$(readlink -f "$0")")`. Does that work for you on Mac?